share/spice/nobel_prize/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+ {{#if hasPrizeCategory}}
+    <span class="one-line text--secondary" >Year : <b>{{year}}</b></span><br/> 
+ {{/if}}
+ 
+ {{#if hasPrizeYear}}
+    <span class="one-line text--secondary" >Category : <b>{{toUpperCase category}}</b></span><br/>
```

Comment:
Because complete lower-case looks very wrong. Ideally capitalizing the first letter should be right.
Thus i have changed the category to look like:
```
Physics
Peace
Medicine
```